src/utils/plotSpecs/generateCellSizeDistributionKneePlot.js

```diff
@@ -133,7 +133,7 @@ const generateSpec = (config, plotData) => {
         scale: 'yscale',
         grid: true,
         zindex: 1,
-        title: { value: config.axes.yAxisText },
+        title: config.axes.yAxisText,
```

Comment:
There seems to be a non-breaking API change for Vega. As seen from this document: https://vega.github.io/vega/docs/title/ the `text` field requires `String`, but we can still use the `value` inside the object. We might need some clean up and updates for this library.

Reply:
I noticed this too, it would be a nice clean-up to make.